Makefile

```diff
@@ -0,0 +1,26 @@
+.PHONY: build-dev build-prod test up-dev up-prod down-dev down-prod
+
+build-dev:
+	@docker-compose -f docker-compose.dev.yml build
```

Comment:
recent versions of docker-compose use `docker compose` without the `-` so this is breaking in my local tests.

Reply:
Docker compose V2 is much better (`docker compose`). It's Golang / much faster & has plugins inbuilt like buildkit.

Reply:
@dakotabenjamin @spwoodcock  gotcha! I'll switch to `docker compose`. thanks!